homeassistant/components/config/label_registry.py

```diff
@@ -0,0 +1,129 @@
+"""Websocket API to interact with the label registry."""
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.components import websocket_api
+from homeassistant.components.websocket_api.connection import ActiveConnection
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.label_registry import LabelEntry, async_get
+
+
+@callback
+def async_setup(hass: HomeAssistant) -> bool:
+    """Register the Label Registry WS commands."""
+    websocket_api.async_register_command(hass, websocket_list_labels)
+    websocket_api.async_register_command(hass, websocket_create_label)
+    websocket_api.async_register_command(hass, websocket_delete_label)
+    websocket_api.async_register_command(hass, websocket_update_label)
+    return True
+
+
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "config/label_registry/list",
```

Comment:
The frontend probably would want to list labels that are used by a specific type (ie automation) to render a filter list. Can be added in a future pr